src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -280,8 +304,54 @@ public static class Parser
         public static ParsedTrackInfo ParseMusicPath(string path)
         {
             var fileInfo = new FileInfo(path);
+            var file = TagLib.File.Create(path);;
+            var albumArtists = file.Tag.AlbumArtists;
+            var album = file.Tag.Album;
+            var trackName = file.Tag.Title;
+            var trackNumber = file.Tag.Track;
+
+            var artistTitleInfo = new ArtistTitleInfo
+            {
+                Title = file.Tag.Title,
+                Year = (int) file.Tag.Year
+            };
+
+            var temp = new int[1];
+            temp[0] = (int) trackNumber;
+            var result = new ParsedTrackInfo
+            {
+                AlbumTitle = file.Tag.Album,
+                ArtistTitle = file.Tag.FirstAlbumArtist, 
+                Quality = QualityParser.ParseQuality(trackName),
+                TrackNumbers = temp,
+                ArtistTitleInfo = artistTitleInfo,
+                Title = file.Tag.Title
+            };
+
 
-            var result = ParseMusicTitle(fileInfo.Name);
+            Logger.Info("Quality parsed: {0}", file.Tag.BeatsPerMinute);
```

Comment:
Also, I would advise moving all such logs to Debug, or else you have a log table full of this stuff after one RSS Sync.